integrations-and-supported-tools/evidently/notebooks/Neptune_Evidently.ipynb

```diff
@@ -0,0 +1,565 @@
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run Evidently test suites and reports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_stability = TestSuite(\n",
+    "    tests=[\n",
+    "        DataStabilityTestPreset(),\n",
+    "    ]\n",
+    ")\n",
+    "data_stability.run(\n",
+    "    current_data=iris_frame.iloc[:60], reference_data=iris_frame.iloc[60:], column_mapping=None\n",
+    ")\n",
+    "data_stability"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_drift_report = Report(\n",
+    "    metrics=[\n",
+    "        DataDriftPreset(),\n",
+    "    ]\n",
+    ")\n",
+    "\n",
+    "data_drift_report.run(\n",
+    "    current_data=iris_frame.iloc[:60], reference_data=iris_frame.iloc[60:], column_mapping=None\n",
+    ")\n",
+    "data_drift_report"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### (Neptune) Start a run\n",
+    "\n",
+    "To connect your script to Neptune and create a new run, we tell Neptune:\n",
+    "* **Who you are** - with a Neptune API token\n",
+    "* **Where to send your data** - to a Neptune project\n",
+    "\n",
+    "The cell below lets you record data to the public project [common/evidently-support](https://app.neptune.ai/o/common/org/evidently-support) as an anonymous user."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import neptune\n",
+    "\n",
+    "run = neptune.init_run(\n",
+    "    api_token=neptune.ANONYMOUS_API_TOKEN,\n",
+    "    project=\"common/evidently-support\",\n",
+    "    tags=[\"reports\"],  # (optional) replace with your own\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Alternatively, you can log the example to your own workspace.\n",
+    "\n",
+    "To do that, replace the code above with the following:\n",
+    "\n",
+    "```python\n",
+    "from getpass import getpass\n",
+    "\n",
+    "run = neptune.init_run(\n",
+    "    api_token=getpass(\"Enter your Neptune API token: \"),\n",
+    "    project=\"workspace-name/project-name\",  # replace with your own\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "For example, if your workspace name is `ml-team` and the project name is `classification`, the project argument is: `project=\"ml-team/classification\"`.\n",
+    "\n",
+    "To find your API token and project name, [log in to Neptune](https://app.neptune.ai/).\n",
+    "- In the top-right corner, click your avatar and select **Get your API token**.\n",
+    "- To find and copy your project name, navigate to the project, then click **Settings** → **Properties**.\n",
```

Comment:
Make sure to update the UI navigation instructions if you're copying from somewhere other than the template.

```suggestion
    "- In the bottom-left corner, expand your user menu and select **Get your API token**.\n",
    "- To copy the project name, in the top-right corner, open the settings menu and select **Properties**.\n",
```